packages/salesforcedx-vscode-core/src/commands/templates/libraryBaseTemplateCommand.ts

```diff
@@ -133,7 +132,7 @@ export abstract class LibraryBaseTemplateCommand<T>
     let customOrgMetadataTemplates;
 
     const configValue = await ConfigUtil.getConfigValue(
-      Config.CUSTOM_ORG_METADATA_TEMPLATES
+      'customOrgMetadataTemplates'
```

Comment:
Config no longer has this prop in v3, so we are using the string to match the key value that users are told in the docs: https://developer.salesforce.com/tools/vscode/en/user-guide/byotemplate#set-default-template-location

Reply:
In a follow up WI (maybe https://github.com/forcedotcom/easywriter/issues/58) this config value should be retrieved using ConfigAggregator and the new key like this: https://github.com/forcedotcom/salesforcedx-templates/pull/503/files#diff-d0896bbf0f0207aa34616bb0c6e0b62c7f1da548dd41da96623e477b6e83203fR59